### PR TITLE
feat: consolidate thread and year group screen reader announcements

### PR DESCRIPTION
**Pull request by assadk88**
## Description

Music year: {owa_music_year}

- Consolidated announcements for both thread and year group selections
- Amended the aria-label for thread radio button's

## Issue(s)

Fixes #[912](https://www.notion.so/oaknationalacademy/Fix-highlighted-units-voice-over-announcements-for-threads-10526cc4e1b18051a4b2f1ce68376b91?pvs=4)

## How to test

1. Go to https://deploy-preview-2947--oak-web-application.netlify.thenational.academy
2. Click on \_\_\_\_\_\_
3. You should see \_\_\_\_\_\_

## Screenshots

N/A

## Checklist

- [ ] ~~Added or updated tests where appropriate~~
- [X] Manually tested across browsers / devices
- [X] Considered impact on accessibility
- [ ] ~~Design sign-off~~
- [X] Approved by product owner
- [ ] ~~Does this PR update a package with a breaking change~~

### CODE REVIEW

**Review by rightsaidjames — CHANGES_REQUESTED** [review]
I've made some suggested improvements to reduce unnecessary verbosity. But other than that I've tested it with VoiceOver and it works perfectly!

File: src/components/CurriculumComponents/UnitsTab/UnitsTab.tsx [hunk 1]
```diff
@@ -351,10 +351,20 @@ const UnitsTab: FC<UnitsTabProps> = ({ trackingData, formattedData }) => {
...
+                {unitCount === 1 ? "unit" : "units"}
+              </p>
+              {selectedThread && (
+                <p>
+                  {highlightedUnitCount()}
+                  {highlightedUnitCount() === 1 ? " unit " : " units "}
+                  highlighted in {selectedThread.title} thread
+                </p>
+              )}
+            </ScreenReaderOnly>
```
[the remark this answers is not in the document]
Comment (assadk88) [reply src/components/CurriculumComponents/UnitsTab/UnitsTab.tsx:367]: Sounds good to me!

**Review by orangemug — APPROVED** [review]

**Review by rightsaidjames — APPROVED** [review]
LGTM!

**Pull request merged**